docs/release-notes/0.11/release-0.11.md

```diff
@@ -0,0 +1,42 @@
+# ML.NET 0.11 Release Notes
+
+[ML.NET](https://aka.ms/mlnet) 0.11 will be the last preview release before we reach `Release Candidate` for v1. We continue to push for creating a coherent and clean API surface for [ML.NET](https://aka.ms/mlnet) users. This release includes several bug fixes as well as extensive work on reducing the public API surface. The work on the API is tracked via [this project](https://github.com/dotnet/machinelearning/projects/13). In the upcoming 0.12 (RC1) releases before we reach 1.0, we will continue on refining the API and improving documentation.
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET
+Core
+2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md)
```

Comment:
couple of line breaks looks weird - is this intentional?

Reply:
We should probably update this to 2.1 since 2.0 is no longer supported....

Reply:
If you consider my big palm intentional! :)